launch/client.py

```diff
@@ -956,6 +966,7 @@ def edit_model_endpoint(
                 min_workers=min_workers,
                 model_bundle_id=model_bundle_id,
                 per_worker=per_worker,
+                high_priority=high_priority,
```

Comment:
Same thing here, with `UpdateModelEndpointRequest` to `UpdateModelEndpointRequestV1`